plugins/imagebase/plugin.js

```diff
@@ -44,8 +54,56 @@
 		return CKEDITOR.plugins.link.parseLinkAttributes( widget.editor, widget.parts.link );
 	}
 
-	var featuresDefinitions = {
-		link: {
+	function getLinkFeature() {
```

Comment:
There are still functions connected with link feature that are outside of this function:

* `isLinkable`
* `addLinkAttributes`
* `createLink`
* `getLinkData`